docs/tutorial/population_dynamics.rst

```diff
@@ -124,55 +173,82 @@ bins to be plotted as `n_bins`.
 This process is repeated many times, and the distribution of the resulting parameters can be analyzed to estimate certain statistics about the them::
 
-    dwell_2.calculate_bootstrap(iterations=1000)
+    dwell_2.calculate_bootstrap(iterations=100)
+
+    plt.figure()
     dwell_2.bootstrap.plot(alpha=0.05)
+    plt.show()
 
-.. image:: kymo_bind_bootstrap_2.png
+.. image:: figures/population_dynamics/dwell2_bootstrap.png
 
-Here we see the distributions of the bootstrapped parameters. The vertical lines indicate the
-means of the distributions, while the red area indicates the estimated confidence intervals. The `alpha` argument determines
-the CI that is estimated as `100*(1-alpha)` % CI; in this case we're showing the estimate for the 95% CI. The values for the
-lower and upper bounds are the `100*(alpha/2)` and `100*(1-alpha/2)` percentiles of the distributions.
+Here we see the distributions of the bootstrapped parameters, each of which ideally should look like a Normal (Gaussian) distribution.
+The vertical lines indicate the means of the distributions, while the red area indicates the estimated confidence intervals.
+The `alpha` argument determines the CI that is estimated as `100*(1-alpha)` % CI; in this case we're showing the estimate for the 95% CI.
+The values for the lower and upper bounds are the `100*(alpha/2)` and `100*(1-alpha/2)` percentiles of the distributions.
 
-Note, however, that while the means correspond well with the optimized model parameters, the distributions are not symmetric.
-In such a case, the simple method of using percentiles as CI values may not be appropriate. For more advanced analysis,
-the distribution values are directly available through the properties `DwelltimeModel.bootstrap.amplitude_distributions` and
-`DwelltimeModel.bootstrap.lifetime_distributions` which return the data as a `numpy` array with
-shape `[# components, # bootstrap samples]`.
+Clearly the distributions here are not Gaussian. Specifically, the two distributions on the left for the fractional amplitudes
+are split, indicating that the model parameters are non-identifiable (there are different combinations of parameters that give
+similar overall results). This analysis strongly indicates that the single exponential model is preferable. We can also look at
+the bootstrap for that model to verify the results are satisfactory::
+
+    dwell_1.calculate_bootstrap(iterations=100)
+
+    plt.figure()
+    dwell_1.bootstrap.plot(alpha=0.05)
+    plt.show()
+
+.. image:: figures/population_dynamics/dwell1_bootstrap.png
+
+Here we only see one distribution since the fractional amplitude for a single exponential model is `1` by definition. The results
+look much better, with most of the distribution being fairly Gaussian with the exception of some outliers at longer lifetimes.
+These likely are the result of poorly fit or numerical unstable models.
```

Comment:
Is it possible to provide custom bounds? If so, it might be nice to show that, because then we can actually see the distribution shape a bit better. Right now, with so few bins squashed on the left side it is difficult to assess.

In the future, it might be nice to be able to omit real failure modes (not for this PR tho).